api/src/api/users.ts

```diff
@@ -132,4 +133,51 @@ router.delete(
   }),
 );
 
+// Add pitch to a user's claimed pitches
+router.put(
```

Comment:
Just had a discussion with @yousefa00 and @arpanlaha about this - while this endpoint structure works, it would be better convention to make this a POST request to `/:userId/pitches` - although you're not creating a new pitch perse, you're creating a new link between a user and a pitch, so POST would make sense in this case. It would also make the endpoint URL slightly shorter!